controller/array_action/array_mediator_ds8k.py

```diff
@@ -652,9 +645,18 @@ def get_flashcopy_state(self, flashcopy_id):
         flashcopy_process = self._get_flashcopy_process(flashcopy_id)
         return flashcopy_process.state
 
-    def _assert_snapshot_is_not_in_use(self, snapshot_id, flashcopies):
+    def _safe_delete_flashcopies(self, api_volume):
+        flashcopies = api_volume.flashcopy
+        flashcopies_as_source = [flashcopy for flashcopy in flashcopies
+                                 if flashcopy.sourcevolume == api_volume.id]
+        for flashcopy in flashcopies_as_source:
+            self._is_flashcopy_safe_to_delete(flashcopy, api_volume.name)
```

Comment:
`is` as 1st word sounds boolean. consider "assert"/"ensure"/etc...

Reply:
done